introduction.md

```diff
@@ -1,3 +1,7 @@
 Hello, I am Moondoobie. I have a degree in Electrical Engineering and an MBA, and I really like learning. I live in the Northwest U.S..
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?